doc/invalidation-handlers.rst

```diff
@@ -9,9 +9,15 @@ to simplify common operations.
 Tag Handler
 -----------
 
-The tag handler helps you to invalidate all cache entries that where marked
-with a specified tag. It works only with a ``CacheInvalidator`` that supports
-``CacheInvalidator::INVALIDATE``.
+.. versionadded:: 1.3
```

Comment:
We use the Read the Docs theme (`theme.css`) on local builds too, so you can see what it will really look like. Unfortunately, [they don’t support the version directives](https://github.com/snide/sphinx_rtd_theme/issues/99). Pushed a commit that at adds some basic formatting for `versionadded/versionchanged`.